app/Http/Resources/RequestProcurementDetailedResource.php

```
@@ -24,6 +24,7 @@ public function toArray(Request $request): array
             "price_quotation_count" => $this->priceQuotations->count(),
             'price_quotations' => PriceQuotationListingResource::collection($this->whenLoaded('priceQuotations')),
             'canvass_summaries' => RequestCanvassSummaryListingResource::collection($this->whenLoaded('canvassSummaries')),
+            'purchase_orders' => RequestPurchaseOrderListingResource::collection($this->whenLoaded('purchaseOrders')),
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Http\Resources\RequestPurchaseOrderListingResource' in method 'toArray'. (Clean Code Rules)

(StaticAccess)